package.json

```diff
@@ -40,5 +40,9 @@
     "lint-staged": "^13.0.3",
     "next": "^14.0.4",
     "vercel": "^32.4.1"
+  },
+  "dependencies": {
+    "chart.js": "^4.4.2",
+    "react-chartjs-2": "^5.2.0"
```

Comment:
I think you bymistake added this dependencies at root of monorepo, we should add this inside `packages/nexts/package.json` dir 

To remove this from here you could run : 

```shell
yarn remove chart.js react-chartjs-2
```

And to add the dependencies in nextjs, you could run : 

```shell
 yarn workspace @se-2/nextjs add chart.js react-chartjs-2
```

Alternatively, you could also first `cd packages/nextjs` and then run ` yarn add chart.js react-chartjs-2` directly

Reply:
Thank you, moved the dependencies!